src/main/java/com/bandwidth/controller/CallbacksController.java

```diff
@@ -30,39 +36,41 @@ public String inboundCall(@RequestBody VoiceCallback callback) {
 
         if("initiate".equalsIgnoreCase(eventType) || "redirect".equalsIgnoreCase(eventType)) {
 
-            Ring ring = Ring.builder().duration(30.0).build();
+            Ring ring = new Ring(30d, true);
 
-            Redirect redirect = Redirect.builder()
+            Redirect redirect = new Redirect().builder()
                 .redirectUrl("/callbacks/inbound")
                 .build();
 
-            response.addAll(ring, redirect);
+            response.withVerbs(ring, redirect);
         }
 
         if ("initiate".equalsIgnoreCase(eventType)) {
             Main.activeCalls.add(callback.getCallId());
         }
 
-        return response.toBXML();
+	logger.info(response.toBxml(jaxbContext));
+
+        return response.toBxml(jaxbContext);
 
     }
 
     @RequestMapping("/goodbye")
```

Comment:
this is `callEnded` in the other samples, we should keep these endpoint names consistent